shell/platform/fuchsia/runtime/dart/utils/BUILD.gn

```diff
@@ -34,10 +34,10 @@ source_set("utils") {
     "$fuchsia_sdk_root/pkg:async-loop-default",
     "$fuchsia_sdk_root/pkg:fdio",
     "$fuchsia_sdk_root/pkg:memfs",
-    "$fuchsia_sdk_root/pkg:syslog",
-    "$fuchsia_sdk_root/pkg:zx",
     "$fuchsia_sdk_root/pkg:sys_cpp",
+    "$fuchsia_sdk_root/pkg:syslog",
     "$fuchsia_sdk_root/pkg:vfs_cpp",
+    "$fuchsia_sdk_root/pkg:zx",
```

Comment:
This was altered by the formatting script. Not sure if I should revert or leave alone.

Reply:
No. Leave it be. This is good. We format all our GN files. I believe this one was imported from //topaz as-is recently.